dashboard/test/models/pd/international_optin_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class Pd::InternationalOptinTest < ActiveSupport::TestCase
+  FORM_DATA = {
+    firstName: 'First',
+    firstNamePreferred: 'Preferred',
+    lastName: 'Last',
+    email: 'foo@bar.com',
+    emailAlternate: 'footoo@bar.com',
+    gender: 'Prefer not to answer',
+    schoolName: 'School Name',
+    schoolCity: 'School City',
+    schoolCountry: 'School Country',
+    ages: ['19+ years old'],
+    subjects: ['ICT'],
+    resources: ['Kodable'],
+    robotics: ['LEGO Education'],
+    workshopOrganizer: 'Workshop Organizer',
+    workshopFacilitator: 'Workshop Facilitator',
+    workshopCourse: 'Workshop Course',
+    optIn: 'Yes'
+  }
+
+  test 'Test international optin validation' do
+    optin = build :pd_international_optin, form_data: {}.to_json
+    refute optin.valid?
+
+    assert build(:pd_international_optin, form_data: FORM_DATA.to_json).valid?
```

Comment:
nit: why 2 lines for refute and one line for assert?

The `optin` variable from line 25 doesn't appear to be used anywhere else (after line 26, which can be combined).

Reply:
Good catch; will simplify.

No particular reason, was just basing it of [these tests](https://github.com/code-dot-org/code-dot-org/blob/394262a0158765dbb4ab4276ea3c1a23455d34e9/dashboard/test/models/pd/regional_partner_contact_test.rb#L25-L28).